app/src/main/java/com/globant/practice/presenter/SplashPresenter.java

```diff
@@ -0,0 +1,71 @@
+package com.globant.practice.presenter;
+
+import com.globant.practice.interfaces.Splash;
+import com.globant.practice.view.activities.SplashActivity;
+
+import java.util.Timer;
+import java.util.TimerTask;
+
+import javax.inject.Inject;
+
+/**
+ * Contains the methods that the SplashActivity needs.
+ * Created by jonathan.vargas on 30/03/2017.
+ */
+
+public class SplashPresenter {
+    Splash splash;
```

Comment:
If this a View, it would be better to give it a more descriptive name like _view_ for instance